pybamm/solvers/base_solver.py

```diff
@@ -589,6 +589,13 @@ def solve(
             for inputs in inputs_list
         ]
 
+        # Cannot use multiprocessing with model in "jax" format
```

Comment:
Note that Jax is designed to support parallel solution of models with different inputs, so this should be doable. But it can be left for a PR later on

Reply:
Okay good, would be better to rely on Jax's native behavior anyway :)